packages/frontend/src/components/header.css

```diff
@@ -53,3 +54,21 @@ input:focus {
   font-size: 1rem;
   padding: 0;
 }
+
+.logo {
+  height: 40px;
+}
+
+.menu {
+  cursor: pointer;
+}
+
+@media only screen and (max-width: 800px) {
+  input {
+    width: 80%;
+  }
+
+  .go {
+    width: 20%;
+  }
+}
```

Comment:
better display for mobile header -- works after shortening ids and Vivi's suggestion https://github.com/Unirep/explorer/pull/45/files#r1250068831:

```
@media only screen and (max-width: 800px) {

  .header {
    padding: 2rem 1.5rem 0.5rem;
  }
  .searchbar {
    padding: 0 2rem;
  }

  input {
    width: 70%;
  }

  .go {
    width: 30%;
  }
}
```